test/filters/not_test.rb

```diff
@@ -35,13 +35,13 @@ def test_to_hash_change_last_term_filter_value
   def test_to_hash_change_last_range_filter_value
     filter_class = MiniTest::Mock.new
     filter_class.expect :valid?, true
-    filter_class.expect :to_hash, {query: {filtered: {filter: {and: {range: {a: {lte: 1}}}}}}}
+    filter_class.expect :to_hash, {query: {filtered: {filter: {and: [{range: {a: {lte: 1}}}]}}}}
```

Comment:
Line is too long. [96/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.